styles/octocat.css

```diff
@@ -0,0 +1,90 @@
+body {
+  margin: 0;
+  font-family: sans-serif;
+}
+
+ul {
+  padding-left: 0;
+  list-style: none;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+img {
+  max-width: 100%;
+}
+
+p {
+  color: rgba(156,156,156,.92);
+  font-size: 0.8rem;
+  font-weight: bolder;
+}
+
+strong {
+  color: black;
+}
+
+/* GRID + general layout */
+
+.section {
+  /*background: yellow;*/
+  padding: 7rem;
```

Comment:
This padding is a bit excessive on mobile devices. For sections when using the double container pattern the padding here is just to keep things slightly off the edge of the page not to make things smaller.

If you want to keep the layout centered and a bit smaller, reduce the max-width on container